scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -446,6 +446,7 @@ workflow CNVSomaticPairWorkflow {
         File copy_ratio_parameters_tumor = ModelSegmentsTumor.copy_ratio_parameters
         File allele_fraction_parameters_tumor = ModelSegmentsTumor.allele_fraction_parameters
         File called_copy_ratio_segments_tumor = CallCopyRatioSegmentsTumor.called_copy_ratio_segments
+        File called_copy_ratio_segments_legacy_tumor = CallCopyRatioSegmentsTumor.called_copy_ratio_legacy_segments
```

Comment:
This should be `called_copy_ratio_legacy_segments_tumor` for consistency.

Reply:
Done